src/shared/mkdir-label.h

```diff
@@ -2,6 +2,7 @@
 #pragma once
 
 #include <fcntl.h>
+#include <sys/stat.h>
```

Comment:
Please include this in mkdir-label.c instead of .h file.

Reply:
Done. thats how I had it but changed it since system headers were included in .h file.

Reply:
yeah done

Reply:
Just to explain: mode_t, uid_t, and gid_t are defined in fcntl.h and sys/types.h. Those types are referenced directly in mkdir-label.h.

The S_IFDIR macro is only referenced in mkdir-label.c; we don't need it in mkdir-label.h.